tests/cupy_tests/lib_tests/test_polynomial.py

```diff
@@ -125,6 +125,20 @@ def test_poly1d_order_leading_zeros(self, xp, dtype):
         a = xp.array([0, 0, 1, 2, 3, 0], dtype)
         return xp.poly1d(a).order
 
+    @testing.for_signed_dtypes()
+    @testing.numpy_cupy_allclose(rtol=1e-6)
+    def test_poly1d_roots1(self, xp, dtype):
```

Comment:
What is the role of this test? It looks able to be merged into `TestRootsReal`.

Reply:
Please don't confuse `poly1d.roots` with `cupy.roots()`. I know it is simply one func calling the other but I thought this call should be tested too

Reply:
`.roots` is just an alias of `cupy.roots`, so we don't need to double-check of this routine except for the typecheck of the return value. Also the test of `poly1d.__add__` is only for typecheck of the return value, and more detailed tests, for example dtype combination, are checked only in `polyadd` test.

Reply:
I know, I didn't repeat `cupy.roots` tests for `.roots` . It is only a single test (taking into consideration that general matrix related tests will be removed). Do you want to omit it?